pkgs/development/tools/unused/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPackages_1_45, cmake }:
+rustPackages_1_45.rustPlatform.buildRustPackage rec {
```

Comment:
Why this particular version of `rustPlatform`?

Reply:
I explained this in [the commit message](https://github.com/NixOS/nixpkgs/pull/107077/commits/d51ff059c956ca9ad3c8974a122a03b7528034bd); do you think it should be in a comment as well?

Reply:
I think so, because comments are generally more durable than commit messages. (Honestly, I didn't think to check commit message, and if code moves it's hard to find the message.)

If this has been reported upstream, an issue link would be neat too.

Reply:
[This reddit comment](https://www.reddit.com/r/rust_gamedev/comments/k0p6i3/amethyst_keeps_erroring/ge7lxn4/?utm_source=reddit&utm_medium=web2x&context=3) is the closest I came to finding a similar report. I'm not even sure what project the bug is in (unused? rustc? nixpkgs?) and don't want to find out.

Reply:
it fails because its internal crate [`project_configuration` uses `rust-yaml`](https://github.com/unused-code/unused/blob/507ac5bf748f10ea25127031eb31e5fe58683004/crates/project_configuration/Cargo.toml#L11) which does not build with rust 1.48.0 https://github.com/chyh1990/yaml-rust/pull/157

Reply:
Thanks @marsam! I've updated the comment with the information you provided.